jetty9-compat-base/src/main/java/com/google/apphosting/vmruntime/jetty9/VmRuntimeWebAppContext.java

```diff
@@ -103,44 +103,47 @@
     System.setProperty(MemcacheSerialization.USE_THREAD_CONTEXT_CLASSLOADER_PROPERTY,
-        Boolean.TRUE.toString());
+      Boolean.TRUE.toString());
   }
 
   // List of Jetty configuration only needed if the quickstart process has been
   // executed, so we do not need the webinf, wedxml, fragment and annotation configurations
   // because they have been executed via the SDK.
   private static final String[] quickstartConfigurationClasses =
-      {org.eclipse.jetty.quickstart.QuickStartConfiguration.class.getCanonicalName(),
-          org.eclipse.jetty.plus.webapp.EnvConfiguration.class.getCanonicalName(),
-          org.eclipse.jetty.plus.webapp.PlusConfiguration.class.getCanonicalName(),
-          org.eclipse.jetty.webapp.JettyWebXmlConfiguration.class.getCanonicalName()};
+    {org.eclipse.jetty.quickstart.QuickStartConfiguration.class.getCanonicalName(),
+      org.eclipse.jetty.plus.webapp.EnvConfiguration.class.getCanonicalName(),
+      org.eclipse.jetty.plus.webapp.PlusConfiguration.class.getCanonicalName(),
+      org.eclipse.jetty.webapp.JettyWebXmlConfiguration.class.getCanonicalName()};
 
   // List of all the standard Jetty configurations that need to be executed when there
   // is no quickstart-web.xml.
   private static final String[] preconfigurationClasses =
-      {org.eclipse.jetty.webapp.WebInfConfiguration.class.getCanonicalName(),
-          org.eclipse.jetty.webapp.WebXmlConfiguration.class.getCanonicalName(),
-          org.eclipse.jetty.webapp.MetaInfConfiguration.class.getCanonicalName(),
-          org.eclipse.jetty.webapp.FragmentConfiguration.class.getCanonicalName(),
-          org.eclipse.jetty.plus.webapp.EnvConfiguration.class.getCanonicalName(),
-          org.eclipse.jetty.plus.webapp.PlusConfiguration.class.getCanonicalName(),
-          org.eclipse.jetty.annotations.AnnotationConfiguration.class.getCanonicalName()};
+    {org.eclipse.jetty.webapp.WebInfConfiguration.class.getCanonicalName(),
+      org.eclipse.jetty.webapp.WebXmlConfiguration.class.getCanonicalName(),
+      org.eclipse.jetty.webapp.MetaInfConfiguration.class.getCanonicalName(),
+      org.eclipse.jetty.webapp.FragmentConfiguration.class.getCanonicalName(),
+      org.eclipse.jetty.plus.webapp.EnvConfiguration.class.getCanonicalName(),
+      org.eclipse.jetty.plus.webapp.PlusConfiguration.class.getCanonicalName(),
+      org.eclipse.jetty.annotations.AnnotationConfiguration.class.getCanonicalName()};
 
   public String getQuickstartWebXml() {
     return quickstartWebXml;
   }
 
+  public VmMetadataCache getMetadataCache() {
+    return metadataCache;
+  }
+
   /**
-   * Set the quickstart WebXml
-   * <p>
-   * If set, this context will not start, rather it will generate the quickstart-web.xml file and
-   * then stop the server. If not set, the context will start normally
+   * Set the quickstart WebXml <p> If set, this context will not start, rather it will generate the
```

Comment:
This was made by the google formatter? Much undesired. Maybe it was confused because of the lack of '.' ideally it should be:
```
/**
 * Set the quickstart Web.xml.
 *
 * <p>If set, this ....
```